src/main/scala/mesosphere/mesos/TaskGroupBuilder.scala

```diff
@@ -582,6 +582,22 @@ object TaskGroupBuilder extends StrictLogging {
     // attach a tty if specified
     container.tty.filter(tty => tty).foreach(containerInfo.setTtyInfo(_))
 
+    container.linuxInfo.foreach { linuxInfo =>
+      val linuxBuilder = mesos.LinuxInfo.newBuilder
+      linuxInfo.seccomp.foreach { seccomp =>
+        val seccompBuilder = mesos.SeccompInfo.newBuilder
+
+        seccomp.unconfined.foreach { unconfined =>
```

Comment:
I was wondering why `seccomp.unconfined` is optional here until I realized that it is the raml type and not the state type (which it should be).
Also, this mix of optionality in RAML/State and Mesos confuses me. Why don't we make it optional like Mesos does? And if there is a good reason to do it, we should _definitely_ stay consistent with our raml/state types.

Reply:
I removed optionality (other than it's enforcement for mesos).   It shouldn't be optional... however it should default to false.... all updated